packages/arb-token-bridge-ui/src/components/TransferPanel/TransferDisabledDialog.tsx

```diff
@@ -67,6 +68,13 @@ export function TransferDisabledDialog() {
     ? getNetworkName(l2ChainIdForTeleport)
     : null
 
+  const isGHO =
```

Comment:
Would this work?
```typescript
const isGHO = selectedToken?.address.toLowerCase() === withdrawOnlyTokens[networks.destinationChain.id]
   ?.find(_token => _token.symbol === 'GHO')
   ?.l1Address.toLowerCase()
```

We don't have to check for Arbitrum One and target it's chain ID key, we can target any `destinationChain.id` and let it be undefined